### PR TITLE
Fix and improve some german strings

### PR DESCRIPTION
**Pull request by U3knOwn**

### CODE REVIEW

File: resources/language/resource.language.de_de/strings.po [hunk 1]
```diff
@@ -1697,12 +1696,12 @@ msgid "CEC Standby"
 msgstr "CEC Standby"
 
 msgctxt "#32800"
-msgid "Überspringe Intro"
-msgstr ""
+msgid "Skipping intro"
+msgstr "Intro überspringen"
```
Comment (pannal) [inline resources/language/resource.language.de_de/strings.po:1700]: Hmm, this is wrong. This is the button saying "I'm going to skip the intro", hence "Skipping intro". "Überspringe Intro" is correct in this case. Same with "Überspringe Abspann".

File: resources/language/resource.language.de_de/strings.po [hunk 2]
```diff
@@ -1762,44 +1761,44 @@ msgstr "Langsame Verbindung"
...
+msgstr "Setzt den Kodi Cache/Puffer readfactor Wert. Standard: 4, Empfohlen: 4-10. Bei aktivem \"Langsame Verbindung\" wird dies automatisch auf 20 gesetzt, da ansonsten der Cache nicht schnell/agressiv genug gefüllt wird."
 
 msgctxt "#32924"
 msgid "Minimize"
 msgstr "Minimieren"
 
 msgctxt "#32925"
 msgid "Playback Settings"
-msgstr "Abspieleinstell."
+msgstr "Wiedergabe-Einstellungen"
```
Comment (pannal) [inline resources/language/resource.language.de_de/strings.po:1800]: This will overflow in the GUI and show "Wiedergabe-Einstellu..."

Comment (U3knOwn) [reply resources/language/resource.language.de_de/strings.po]: Resolved!

File: resources/language/resource.language.de_de/strings.po (hunk 1 above)
Comment (U3knOwn) [reply resources/language/resource.language.de_de/strings.po]: Resolved!

**Comment by pannal** [comment]
Thanks, will be included in the next release!

**Pull request merged**